src/library/Uno.Material/Styles/Controls/CheckBox.xaml

```diff
@@ -351,6 +254,222 @@
 												Value="1" />
 									</VisualState.Setters>
 								</VisualState>
+
+
+								<VisualStateGroup.Transitions>
+									<VisualTransition From="UncheckedNormal"
+													  To="UncheckedPressed">
+										<xamarin:Storyboard>
```

Comment:
why xamarin: ?

Reply:
this was present previously I will ask @MaximeDion-Work its use.

Reply:
@MatFillion Maxime verified and in wasm the animations do not work (though they don't break either) so I can remove the xamarin and open an issue

Reply:
Ok lets do that! Thanks @NicolasChampagne